web/book/src/project/integrations/postgresql.md

```diff
@@ -0,0 +1,86 @@
+# PostgreSQL
+
+PL/PRQL is a PostgreSQL extension that lets you write functions with PRQL. For example:
+
+```sql
+create function player_stats(int) returns table(player text, kd_ratio real) as $$
+  from rounds
+  filter match_id == $1
+  group player (
+    aggregate {
+      total_kills = sum kills,
+      total_deaths = sum deaths
+    }
+  )
+  filter total_deaths > 0
+  derive kd_ratio = total_kills / total_deaths
+  select { player, kd_ratio }
+$$ language plprql
+
+```
+
+You can also pass PRQL code to the `prql` function. For example:
+
+ ```
+ select prql('from rounds | filter match_id == 1, 'rounds_cursor');
+ ```
+
+For more information on the extension, see the [PL/PRQL repository](https://github.com/kaspermarstal/plprql).
+
+# Getting started
+On Ubuntu, follow these steps to install PL/PRQL from source:
+
+1. Install `cargo-pgrx`.
+
+    ```cmd
+    cargo install --locked --version=0.11.2 cargo-pgrx
+    ```
+
+    The version of `cargo-pgrx` must match the version of `pgrx` in `Cargo.toml`.
+
+2. Initialize `pgrx` for your system.
+   ```cmd
+   cargo pgrx init --pg16 <PG16>
+   ```
+   where `<PG16>` is the path to your system installation's `pg_config` tool (typically `/usr/bin/pg_config`). Supported versions are PostgreSQL v12-16. You can also run `cargo pgrx init` and have `pgrx` download, install, and compile PostgreSQL v12-16. These installations are managed by `pgrx` and used for development and testing. Individual `pgrx` installations can be installed using e.g. `cargo pgrx init --pg16 download`.
+
+3. Clone this repository and `cd` into root directory.
```

Comment:
Agreed, let's link to the README for installation instructions. Would also reduce the maintenance burden for merging PRs here every time changes/additions are made.

Can you be more specific on what you want to adjust? I appreciate the feedback.

Reply:
Great!

> Can you be more specific on what you want to adjust?

No need for changes if we link rather than embed the Readme. 

If we embed the whole Readme, I had suggested we change things that refer to "this repository" to refer to the `plprql` repository...